apps/ledger-live-mobile/src/analytics/index.ts

```diff
@@ -1,7 +1,5 @@
-// eslint-disable-next-line import/no-cycle
 import Track from "./Track";
-// eslint-disable-next-line import/no-cycle
 import TrackScreen from "./TrackScreen";
-// eslint-disable-next-line import/no-cycle
+
```

Comment:
Unrelated to the goal of this PR but I managed to get rid of the require cycle 🧹